lib/routes-priv.js

```diff
@@ -431,6 +431,50 @@ class RoutesPriv {
     })
   }
 
+  async onPrivatePushTxHttpRequest (req, res) {
+    const rpc = this.mainnetRpc
+
+    const payload = req.body
+    const _v = Joi.validate(payload, schemaMainTx)
```

Comment:
As I see [schemaMainTx](https://github.com/bitfinexcom/moonbeam/pull/53/files#diff-aacdac560145b2e6e58f5d1fea0b0821R23) shouldn't have .single() method I think, based on [docs](https://github.com/hapijs/joi/blob/master/API.md#arraysingleenabled) I see that it accepts also non array values, but [push transaction](https://developers.eos.io/manuals/eos/latest/nodeos/plugins/chain_api_plugin/api-reference/index#operation/push_transaction) requires that `signatures` field to be array

Reply:
makes sense, will correct that